packages/components/src/components/RunAction/RunAction.js

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2019-2021 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { injectIntl } from 'react-intl';
+import { Button } from 'carbon-components-react';
+import { Play32 as Play, Restart32 as Restart } from '@carbon/icons-react';
```

Comment:
Let's use PlayOutline32 here, this is more visually consistent with other icons already used and also consistent with use of some of these components in other applications.